### PR TITLE
Show error if rendering RDX `BrowserView` fails

### LINKED ISSUE #4275: Show an error page when extension fails to load

### PR DESCRIPTION
**Pull request by rak-phillip**
This renders an error in page when the if the extension `BrowserView` fails to load.

## Screenshot

image

closes #4275

### CODE REVIEW

**Review by mook-as — COMMENTED** [review]
I didn't do a full review, I was just here looking for a screen shot :D

File: pkg/rancher-desktop/pages/extensions/_root/_src/_id.vue [hunk 1]
```diff
@@ -2,7 +2,15 @@
 import { ipcRenderer } from 'electron';
 import Vue from 'vue';
 
+import { hexDecode } from '@pkg/utils/string-encode';
+import ExtensionsError from '~/components/ExtensionsError.vue';
```
[the remark this answers is not in the document]
Comment (rak-phillip) [reply pkg/rancher-desktop/pages/extensions/_root/_src/_id.vue]: Good catch. The dreaded auto-import strikes again.

File: pkg/rancher-desktop/pages/extensions/_root/_src/_id.vue [hunk 2]
```diff
@@ -2,7 +2,15 @@
...
+
+interface ExtensionsData {
+  error: Error | undefined;
```
[the remark this answers is not in the document]
Comment (rak-phillip) [reply pkg/rancher-desktop/pages/extensions/_root/_src/_id.vue:9]: An error key will always be present value. I think that `error: Error | undefined` expresses this better.

**Review by ericpromislow — APPROVED** [review]
Looks good. The diff here reports some problems with an unused `os` import, but that looks outdated.

**Pull request merged**